test/todo_test.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:clean_todo/data/local/local_storage.dart';
+import 'package:clean_todo/data/local/todo/todo_local_service.dart';
+import 'package:clean_todo/data/repository_impl/todo/todo_repository_impl.dart';
+import 'package:clean_todo/domain/entity/todo_entity.dart';
+import 'package:clean_todo/domain/repository/todo/todo_repository.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+void main() {
+  group("TodoRepositoryImpl", () {
+    test("Adding Todo to storage, length should be 1", () async {
+      SharedPreferences.setMockInitialValues({});
+      SharedPreferences localService = await SharedPreferences.getInstance();
```

Comment:
Why are you using the shared preference directly, use the wrapper class I made for the shared pref, 

The LocalStorage class is a singleton for SharedPreference you don't have to always await it and get the preference